build_tools/python/e2e_test_framework/models/model_groups.py

```diff
@@ -5,6 +5,24 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 """Defines the groups of models."""
 
-from e2e_test_framework.models import tflite_models
+from e2e_test_framework.models import tf_models, tflite_models
 
-MOBILE = [tflite_models.MOBILENET_V2]
+# Small models on mobile devices.
```

Comment:
Aren't we currently running some of these models on desktop also?

Reply:
Removed the MOBILE term.